spec/workers/data_update_worker_spec.rb

```diff
@@ -13,7 +13,7 @@
   it "runs scripts that need running" do
```

Comment:
I added a new data update script file in the fixtures that will produce a failure. Hence I've updated the tests to now cater for two data update scripts and for both the successful and failing data update scripts.